app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -358,7 +358,7 @@ public void ensureHabitOnHomeScreen(){
         _solo.clickOnView(_solo.getView(R.id.add_habit_fab));
 
         // Create test habit
-        Habit testHabit = new Habit("addHabitSuccessTest", "Test Reason", new Date(), getCurrentDay(), PUBLIC_HABIT, HABIT_ID, EMPTY_HABIT_EVENT_LIST);
+        Habit testHabit = new Habit("habitHomeScreenTest", "Test Reason", new Date(), getCurrentDay(), PUBLIC_HABIT, HABIT_ID, EMPTY_HABIT_EVENT_LIST);
```

Comment:
I also went in and changed the habit titles for all the UI tests. Often what happens is, one will fail, and the other tests used the same name so the asserts fail because there is still a habit with that title remaining in the habit list at the end of the test.